hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/util/S3StorageType.java

```diff
@@ -62,6 +62,10 @@ public ReplicationType getType() {
   public static S3StorageType getDefault(ConfigurationSource config) {
     String replicationString = config.get(OzoneConfigKeys.OZONE_REPLICATION);
     ReplicationFactor configFactor;
+    if (replicationString == null) {
```

Comment:
You can check for `StringUtils.isEmpty(replicationString)` to handle even if config is present without value.

Reply:
I think empty value (`""`) is similar to any other invalid value (e.g. `"asdf"`), and should be handled the same way.  So `== null` check is better here.